EntraIdUsersSupportQuickstart/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Azure.Identity;
+using Azure.Communication;
+
+namespace EntraIdUsersSupportQuickstart
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            Console.WriteLine("Azure Communication Services - Obtain Access Token for Entra ID User Quickstart");
+
+            // This code demonstrates how to fetch your Microsoft Entra client ID and tenant ID from environment variables.
+            string clientId = Environment.GetEnvironmentVariable("ENTRA_CLIENT_ID");
+            string tenantId = Environment.GetEnvironmentVariable("ENTRA_TENANT_ID");
+
+            //Initialize InteractiveBrowserCredential for use with CommunicationTokenCredential.
+            var options = new InteractiveBrowserCredentialOptions
+            {
+                TenantId = tenantId,
+                ClientId = clientId,

```

Comment:
We don't need the redirectUri because there's a default value, right?

Reply:
it was an optional param, though for a custom clientID should be defined, so added
![image](https://github.com/user-attachments/assets/62c7978b-bab3-4285-8911-0763529d51b7)